README.md

```diff
@@ -1,9 +1,8 @@
 ![Logo](https://keylighter.kadet.net/img/logo.png)
 
 [![Packagist](https://img.shields.io/packagist/v/kadet/keylighter.svg?style=flat-square)](https://packagist.org/packages/kadet/keylighter)
-[![Scrutinizer Code Quality](https://img.shields.io/scrutinizer/g/kadet1090/keylighter.svg?style=flat-square)](https://scrutinizer-ci.com/g/kadet1090/KeyLighter/?branch=master)
-[![Travis build](https://img.shields.io/travis/kadet1090/KeyLighter.svg?style=flat-square)](https://travis-ci.org/kadet1090/KeyLighter)
-[![Code Coverage](https://img.shields.io/scrutinizer/coverage/g/kadet1090/keylighter.svg?style=flat-square)](https://scrutinizer-ci.com/g/kadet1090/KeyLighter/?branch=master)
+[![GitHub Actions](https://github.com/kadet1090/keylighter/actions/workflows/tests.yml/badge.svg?event=push)](https://github.com/kadet1090/KeyLighter/actions?query=branch:master)
+[![GitHub Actions](https://github.com/kadet1090/keylighter/actions/workflows/phpstan.yml/badge.svg?event=push)](https://github.com/kadet1090/KeyLighter/actions?query=branch:master)
 [![Try it](https://img.shields.io/badge/www-try%20it-FF9700.svg?style=flat-square)](https://keylighter.kadet.net)
```

Comment:
OCD-driven comment: badges for GitHub actions don't support a "flat" style so we might wanna change the other badges to not be flat anymore, for the sake of consistency.